File: JavaScriptBasics/ConditionalStatements/Exercise/Shopping.js
function Shopping(input){
    let budget = Number(input[0])
    let videocards = Number(input[1]);
    let processors = Number(input[2]);
    let ram = Number(input[3]);

    let total = (videocards*250)+(processors*0.35*videocards*250)+(ram*0.1*250*videocards);

    if(processors>videocards){
       if(budget>total){
        console.log("You have "+(budget-total).toFixed(2)+ " leva left!");
       }else{
        console.log("Not enough money! You need "+(total-budget).toFixed(2)+ " leva more!");
       }
    }else{
        if(budget>total){
            console.log("You have "+((budget-total*0.85)).toFixed(2)+ " leva left!");
           }else{
            console.log("Not enough money! You need "+((total-budget)*0.85).toFixed(2)+ " leva more!");
           }
    }
}

File: JavaBasics/ConditionalStatements/Exercise/Shopping.java
package EntryModule.ConditionalStatements.Exercise;

import java.util.Scanner;
public class Shopping {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        double budget = Double.parseDouble(scanner.nextLine());
        int videocards = Integer.parseInt(scanner.nextLine());
        int processors = Integer.parseInt(scanner.nextLine());
        int ram = Integer.parseInt(scanner.nextLine());

        int totalvideo = videocards*250;
        double totalprocessors = (totalvideo*0.35)*processors;
        double totalram = (totalvideo*0.1)*ram;
        double total = totalvideo+totalprocessors+totalram;

        if(videocards>processors){
            total=total*0.85;
        }

        if(total>budget){
            System.out.printf("Not enough money! You need %.2f leva more!", total-budget);

        }
        if(budget>=total){
            System.out.printf("You have %.2f leva left!", budget-total);
        }







    }
}
